src/scheduler/job_info.c

```diff
@@ -5385,12 +5382,12 @@ static char **parse_runone_job_list(char *depend_val) {
 	char *start;
 	const char *depend_type = "runone:";
 	int i;
-	int len = DEFAULT_RUNONE_JOBS;
+	int len = 1;
```

Comment:
Why does len start at one?
Won't there be a colon for every job in the list?

Reply:
since depend_type now has a colon, first colon is skipped when I do - 
r = start + strlen(depend_type);

Reply:
Oops, I must have missed that!